build/debian-hyperkube-base/Makefile

```diff
@@ -23,7 +23,7 @@ TAG=0.6
 ARCH?=amd64
```

Comment:
it seems like the TAG bump here got lost :\

Reply:
sorry, so many bumps I forgot to bump it an extra time. fixed!

Reply:
seems like the bump in `cluster/images/hyperkube/Makefile` got lost, too.